### PR TITLE
Update expo example

### PR DESCRIPTION
**Pull request by Jacse**
This PR deletes the old expo example and moves the code to a newly initialised expo example folder, `example`,  which is already covered in `.npmignore`. Testing and launching is now as easy as cloning and running `yarn start` - no need to install expo globally.

This move does not require a minor version bump, a patch will suffice.

Awaiting approval from @markusl

### CODE REVIEW

File: example/App.js [hunk 1]
```diff
@@ -71,15 +68,16 @@ export default class ProgressChart extends React.Component {
           backgroundWidth={5}
           fill={fill}
           tintColor="#00e0ff"
-          onAnimationComplete={() => console.log('onAnimationComplete')}
```
Comment (markusl) [inline example/App.js]: I'd like to keep this as part of example since it is useful to see that the feature actually works while running the app

Comment (Jacse) [reply example/App.js]: Cool with me 👍 Any other thoughts?

File: example/app.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "expo": {
+    "sdkVersion": "22.0.0"
```
Comment (markusl) [inline example/app.json]: The latest version seems to be 23

**Comment by markusl** [comment]
Looking good! If you could rebase on the latest master and fix the comments it can be merged

**Comment by Jacse** [comment]
Done - I'll let you merge.

**Pull request merged**